tools/bliss-to-ogg-zip.py

```diff
@@ -366,19 +368,25 @@ def main():
       if os.path.exists(dest_filename):
         print("already exists, delete: %s" % os.path.basename(dest_filename))
         os.remove(dest_filename)
-      cmd = ["ffmpeg"]
-      if args.ffmpeg_acodec:
-        cmd += ["-acodec", args.ffmpeg_acodec]  # https://trac.ffmpeg.org/ticket/2810
-      cmd += ["-i", source_filename]
-      if args.number_of_channels > 0:
-        cmd += ["-ac", str(args.number_of_channels)]
-      if start_time:
-        cmd += ["-ss", str(start_time)]
-      if limit_duration:
-        cmd += ["-t", str(duration)]
-      cmd += [dest_filename, "-loglevel", args.ffmpeg_loglevel]
-      print("$ %s" % " ".join(cmd))
-      check_call(cmd)
+      if args.no_conversion:
```

Comment:
Shouldn't there be a check like
`assert rec_filename.endswith(".ogg")`?

Reply:
We could also support any file ending, but I will go with the assert for now.